docs/reference/mapping/types/unsigned_long.asciidoc

```diff
@@ -2,7 +2,7 @@
 [testenv="basic"]
 
 [[unsigned-long]]
-=== Unsigned long data type
+=== Unsigned long
```

Comment:
This should make the title consistent with other field types.
```suggestion
=== Unsigned long field type
++++
<titleabbrev>Unsigned long</titleabbrev>
++++
```

This page should probably be included with the other numeric types, but I can handle that as a separate change.